pkgs/agenix.sh

```diff
@@ -13,7 +13,7 @@ function show_help () {
   echo '-h, --help                show help'
   # shellcheck disable=SC2016
   echo '-e, --edit FILE           edits FILE using $EDITOR'
-  echo '-r, --rekey               re-encrypts all secrets with specified recipients'
+  echo '-r, --rekey [PUBLIC_KEY]  re-encrypts all secrets with specified recipients'
```

Comment:
Not sure if this makes sense here.

It makes parsing harder e.g. `--rekey -v` is currently broken.